tutorials/time-freq/ssvep.py

```diff
@@ -0,0 +1,359 @@
+"""
+==================================
+Basic analysis of an SSVEP dataset
+==================================
+
+Example script to compute frequency spectrum and extract snr of a target frequency
+
+We use a simple example dataset with frequency tagged visual stimulation (a.k.a.
+steady state visually evoked potentials - SSVEP):
+
+N=2 participants observed checkerboards patterns inverting with a constant frequency
+of either 12Hz of 15Hz. 10 trials of 30s length each. 32ch wet EEG was recorded.
+
+Data format: BrainVision .eeg/.vhdr/.vmrk files organized according to BIDS standard.
+
+Data can be downloaded at https://osf.io/7ne6y/
+"""  # noqa: E501
```

Comment:
this `# noqa E501 ` appears in some places. i guess its an artefact?

Reply:
`# noqa E501` tells code style checkers (flake8 in the case of mne-python) that they should ignore style error "E501" which is "line is too long". I don't think it does anything here though. I'll remove it and see if flake8 will complain about this line.

Reply:
Flake8 did complain when I removed the `noqa` hint. The problem is that some lines of the multiline string that ends with `"""  # noqa: E501` are too long. And you can't put comments within such a string, so you just put one `noqa` in the end. This is done in some other tutorials as well, so I'll keep `noqa` here instead of reformatting the text.

Reply:
The other place with `noqa` poses a bit of a problem. See comments in [another conversation](https://github.com/dominikwelke/mne-python/pull/1#discussion_r459541141).